spec/forms/ctc/spouse_filed2019_form_spec.rb

```diff
@@ -0,0 +1,87 @@
+require "rails_helper"
+
+describe Ctc::SpouseFiled2019Form do
+  let(:client) { create :client, tax_returns: [(create :tax_return, filing_status: nil )]}
+  let!(:intake) { create :ctc_intake, client: client }
+  let(:params) { { spouse_filed_2019: filed_2019 } }
+  let(:filed_2019) { "filed_full_joint" }
+
+  context "validations" do
+    context "when filing status is selected" do
+      it "is valid" do
+        expect(
+            described_class.new(intake, params)
+        ).to be_valid
+      end
+    end
+
+    context "when spouse_filed_2019 is not filled in" do
+      it "is not valid" do
+        expect(
+            described_class.new(intake, {})
+        ).not_to be_valid
+      end
+    end
+
+    context "when spouse_filed_2019 is not in the set" do
+      let(:filed_2019) { "on_the_moon" }
+      it "is not valid" do
```

Comment:
Consider `it "is valid at the form level but invalid at the model level" do`

to explain this behavior.

Not a blocker.

Reply:
If the model has testing for this validation, then consider removing this test.